lutris/gui/lutriswindow.py

```diff
@@ -439,8 +443,14 @@ def _open_browser(url):
 
     @GtkTemplate.Callback
     def on_resize(self, widget, *args):
-        """WTF is this doing?"""
-        self.window_size = widget.get_size()
+        """size-allocate signal
+        Updates stored window size and maximized state
```

Comment:
Should have a blank line separating this from the method "title". Also, proper grammar: end with a period/`.`.

https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings